spk/python3/Makefile

```diff
@@ -9,6 +9,8 @@ DEPENDS += cross/setuptools cross/pip cross/wheel cross/virtualenv cross/six
 DEPENDS += cross/bcrypt cross/pyopenssl cross/pycrypto
 DEPENDS += cross/pycurl cross/sqlalchemy cross/lxml
 DEPENDS += cross/zope.interface cross/markupsafe
+DEPENDS += cross/pyyaml
+#DEPENDS += cross/netifaces cross/pyephem
 
```

Comment:
If you're preparing for a new version by bumping SPK_REV, please add/update the CHANGELOG entry too.